### ISSUE TITLE
Changing the resources of buildings 

### ISSUE THREAD

**Issue by Megamonster42** [Sh0wdown/UnofficialCrusaderPatch#476]
Is it possible to change the costs of some buildings ?

**Comment by J-T-de** [comment]
Short answer: Yes!
(The patch already changes the price of the market (to zero)

Any more specific thoughts (like implementable for all patch users) or more for personal taste? For example, blacksmith costs were discussed in https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/469, and for example I recall somebody suggesting to make also the houses free, as the AI sometimes struggles when it has wood shortages. Or changing unit costs, or...

Anyway, except for the free market and the costs of Ladderman, no "finetuning" of ingame cost was done in the patcher and always leads to discussions.

P.S.: I suggest to change the title, as this is quite misleading: It suggests the question "Is it possible to change the resources a building produces?" (Yes, amount and resource should also be possible (If one wants a stone producing Fletcher).)

**Comment by Megamonster42** [comment]
The thing is i want to create an alternative winning scenario for my offline LAN event. 
For this I thought about changing the costs of the round tower to (1000 wood and 1000 Stone). 
Sorry if my question was not that clear. 
If this is possible how can I do it ?

**Comment by PodeCaradox** [comment]
I can maybe look sunday into the round tower to change it, maybe I can give you a script or the changed exe.

**Comment by J-T-de** [comment]
Yeah, maybe I should make my answer more clear: It is in general possible, however, the Patch as it is right now does not have this as an option.

**Comment by PodeCaradox** [comment]
I can look into it with cheatengine, and maybe I can change it

**Comment by Megamonster42** [comment]
J-T-de that's true and I know it. 
PodeCaraox did you find out something ? Or can you tell me more about the cheatengine, so I can give it a try ?

**Comment by J-T-de** [comment]
Okay, I found all building costs (and the corresponding buildings) in the .exe, I only have to figure out how to include it to the UCP. Single costs would easily work (as depicted here https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/270 ), but I would prefer an .aic - like config, where I have to figure out how to implement this.

Regarding Cheat-Engine: here are the basics one has to know https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/270#issuecomment-475943521 , but this alone is only the start...

**Comment by Krarilotus** [comment]
damn, if @J-T-de is as quick as that, we might see a new realease of the UCP just by him soon. Maybe you ll take over his work and just contribute. I am sure, if he sees your skills he ll make you a contributer and then you can manage the project alongside him somehow?
